### PR TITLE
New frontend

### PR DESCRIPTION
**Pull request by donpaul999**
- Change the frontend part

- Comments are visible

- Website is responsive

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -34,17 +34,21 @@
 @WebServlet("/data")
```
Comment (ramaroberto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Avoid these kind of changes even if minimal, they confuse the reviewer if the title only specifies frontend changes.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -66,3 +63,46 @@ function createListElement(text) {
...
+
+  if (elem1.style.display === "none") {
+    elem1.style.display = "block";
+    elem2.style.display = "none";
+    elem3.style.display = "none";
+  } else {
+    elem1.style.display = "none";
+  }
+
+}
```
Comment (ramaroberto) [inline portfolio/src/main/webapp/script.js]: You're duplicating code here, you've 2 options:

1. Write a function that hides all the elements and re-use it while showing the correct element afterwards.

2. Write a function with a parameter that specifies which element to display, then you can call it from the HTML directly by binding it to the element by doing, for example: ``onClick="showSection('contact')"``

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,35 +1,49 @@
...
-            <ul class="list-group" id="messages-container"></ul>
+  <body style="background-color: #435969;">
+    <div id="content">
+     <div id= "header">
+      <img src="img/picture_of_me.jpg" alt="Paul" style="width:300px;height:180px;">  
+      <h1></h1>
+      <p>Hi! I'm a <b>STEP Intern</b> in the Plx ZRH team. I'm from Romania, a country from the Southestern part of Europe. </br>
+      <div id="buttonline">
+        <button onclick="showAbout()">About Me</button>
+        <button onclick="showSkills()"> Skills </button>
```
Comment (ramaroberto) [inline portfolio/src/main/webapp/index.html]: ``<button onclick="showSkills()"> Skills </button>``

Remove the spaces between ``> Skills <``, it should be

``<button onclick="showSkills()">Skills</button>``

Some for the other tags in this file.

**Review by ramaroberto — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (donpaul999) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: Solved

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (donpaul999) [reply portfolio/src/main/webapp/index.html]: Solved

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (donpaul999) [reply portfolio/src/main/webapp/script.js]: Solved

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -42,3 +39,19 @@ function createListElement(text) {
   liElement.classList.add('list-group-item');
   return liElement;
 }
+
+function showSection(idToShow) {
+  var elem1 = document.getElementById("about");
+  var elem2 = document.getElementById("skills");
+  var elem3 = document.getElementById("contact");
+  var elem4 = document.getElementById(idToShow);
+  var statusOfElem4 = elem4.style.display;
```
Comment (ramaroberto) [inline portfolio/src/main/webapp/script.js:48]: nit: line break after the variable definition

**Review by ramaroberto — CHANGES_REQUESTED** [review]
Feel free to merge after solving the small nit.

**Pull request merged**